modules/diskq/qdisk.c

```diff
@@ -597,15 +597,12 @@ _load_state(QDisk *self, GQueue *qout, GQueue *qbacklog, GQueue *qoverflow)
     {
       if (!_load_non_reliable_queues(self, qout, qbacklog, qoverflow))
         return FALSE;
-    }
 
-  if (!self->options->read_only)
-    {
-      end_ofs = qdisk_try_to_truncate_file_to_minimal(self);
-    }
+      if (!self->options->read_only)
```

Comment:
Originally this block was called in both reliable and nonreliable case. Now only nonreliable case. Is this behavior change intended?

Reply:
Yes.
If you look the previous code it was called on reliable queues as well, but
since it started by checkout qout_ofs, which was 0 in case of reliable, we never truncated in that case.
I think it's much cleaner now.